ictv/renderer/preview.html

```diff
@@ -19,6 +19,8 @@
     along with ICTV.  If not, see <http://www.gnu.org/licenses/>.
 #}
 
+{# If controls is not defined, it must be True by default #}
+{% set controls = (False if controls == False else True) %}
```

Comment:
There is a also a ``default`` filter that may be more useful, but not sure it actually is.